src/lib/workspace/Subsetting/SubsetDownloadModal.tsx

```diff
@@ -223,7 +229,7 @@ export default function SubsetDownloadModal({
   >['serverSidePagination']['fetchPaginatedData'] = useCallback(
     ({ pageSize, pageIndex, sortBy }) => {
       if (!currentEntity) return;
-      if (selectedVariableDescriptors.length === 0) {
+      if (selectedVariableDescriptors.length === 0 || !canLoadTablePreview) {
```

Comment:
`canLoadTablePreview` should probably be added to the list of dependencies for this callback function (https://github.com/VEuPathDB/web-eda/blob/f606310611cbeda0bfa01d30b441e8bcb981053e/src/lib/workspace/Subsetting/SubsetDownloadModal.tsx#L280-L287)

Reply:
Right, because if this value changed we'd definitely want to stop/start fetching data. Thank you!